content/SCALE/SCALETutorials/Apps/AddStorjNode.md

```diff
@@ -1,168 +1,153 @@
+{{< trueimage src="/images/SCALE/22.12/InstallApplicationsStorjSCALE.png" alt="Install Storj App SCALE" id="4 Install Storj App SCALE" >}}
 
-Next, select the checkbox **Configure Identity Volume for Storage Node**. Browse to the host path by clicking on the arrow <span class="material-icons">chevron_right</span> next to the System Dataset Pool icon. A dropdown navigation menu appears. Select the newly created dataset (in our example: identity). This path appears in the **Host Path for Storj Identity Volume** field.
-
-By default, the **Extra Host Path Volumes** field is empty. Click the **Add** button to define a **Mount Path in Pod** and a **Host Path**.
-
-
-![InstallStorjAppStorageExtraHostPathSCALE](/images/SCALE/22.12/InstallStorjAppStorageExtraHostPathSCALE.png "Storage Extra Host Path Storj App SCALE")
-
-### Network Settings
-
-
-![InstallStorjAppNetworkingSCALE](/images/SCALE/22.12/InstallStorjAppNetworkingSCALE.png "Network Settings Storj App SCALE")
-
-Enter the web port in the **Web Port for Storj** field. In this example we are keeping the default setting `20909`.
+1. Accept the default name or enter a new name for your Storj application.
+{{< hint type=note >}}
+You can enter a name for the Storj app using lowercase alphanumeric characters that begin and end with an alphanumeric characters. 
+Do not use a hyphenas the first or last character. For example, *storjnode*, or *storj-node*, but not *-storjnode* or *storjnode-*. 
+{{< /hint >}}
 
-Enter the node port in the **Node Port for Storj** field. In this example, we are keeping the default setting `20988`.
+{{< trueimage src="/images/SCALE/22.12/InstallStorjAppNameSCALE.png" alt="Name Storj App SCALEL" id="5 Name Storj App SCALE" >}}
 
-### Advanced DNS Settings
+2. [Enter the wallet address](#getting-a-wallet-address) in **Configure Wallet for Storj**.
 
+{{< trueimage src="/images/SCALE/22.12/InstallStorjAppConfigSCALE.png" alt="Config Storj App SCALE" id="6 Config Storj App SCALE" >}}
 
-![InstallStorjAppAdvDNSSCALE](/images/SCALE/22.12/InstallStorjAppAdvDNSSCALE.png "Advanced DNS Settings Storj App SCALE")
+3. Enter the [authentication token copied from Storj](#generating-an-authentication-token-for-storj) into **Configure Auth token for Storj Node** and then the email address you associated with the token in **Configure Email for Storj**.
 
-Click the **Add** button to create advanced DNS settings. In this example, we are keeping the default settings and not entering any advanced DNS settings.
+4. Enter the storage domain (i.e., the public network DNS name) added for Storj in **Add Your Storage Domain for Storj**. 
+If using Dynamic DNS (DDNS), enter that name here as well. For exampl, *name.ddns.net*.
 
-### Resource Limits
+5. Accept the default values in **Owner User ID** and **Owner Group ID**. 
 
+6. Configure the storage size you want to share. Enter the value in GBs in **Configure Storage Size You Want to Share in GB's**.
```

Comment:
lot of ins here...

```suggestion
6. Configure the storage size you want to share. Enter the value in **Configure Storage Size You Want to Share in GB's**.
```

Reply:
This is tricky because we want the user to know to enter a value in Gb or a GB value in the field that says to enter it in GB's... I hate that 's in the UI. I'll reword this.